pkg/get/download.go

```diff
@@ -31,6 +32,24 @@ func Download(tool *Tool, arch, operatingSystem, version string, downloadMode in
 		return "", "", err
 	}
 
+	// Check user provided version is valid on dry run
+	if len(version) > 1 && dry_run {
+		res, err := http.Head(downloadURL)
+		if err != nil {
+			return "", "", err
+		}
+
+		if res.StatusCode != http.StatusOK {
+			return "", "", fmt.Errorf("incorrect status for downloading tool: %d", res.StatusCode)
+		}
+
+	}
+
+	if dry_run {
+		fmt.Printf("Download URL: %s\n", downloadURL)
+		os.Exit(0)
```

Comment:
I would not expect to see an os.Exit within a package, if the Github diff is working right, we are looking at pkg/get/download.go

A return statement would make more sense.